README.md

```diff
@@ -1,5 +1,19 @@
 # Forklift
 
+ * [Vagrant Deployment](#vagrant-deployment)
+ * [Direct Deployment](#direct-deployment)
```

Comment:
I know this terminology has been around for a while, but could we rename this to 'Non-vagrant deployment'